source/documentation/submit-message.html.md.erb

```diff
@@ -15,6 +15,14 @@ The X-Message-Type header **must** be the same as in the message body.
 
 The consignor ID or consignee ID must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
 
+## Path parameters
+
+Parameters must be entered in lowercase.
+
+| Parameter | Type | Details |
+| -- | -- | -- |
+| lrn | String | Filter messages by local reference number (LRN) |
```

Comment:
The details of this should probably be something like "The local reference number of the movement"

Reply:
Thanks